### DISCUSSION TITLE
Bot always mention the question.

### DISCUSSION THREAD

**Discussion by IT-Maxy** [Zain-ul-din/whatsapp-ai-bot#2] [Q&A] [answered]
Hello

I'm testing your bot.
was a smooth installation with no error, and everything working as expected.
but have a question:
when bot replies, always do a mention to your question.
is that from your bot or is from any external library?
how do you change it

**Comment by Zain-ul-din** [comment, marked as answer]
Thank you for testing the bot and I'm glad to hear that everything is working as expected. It seems like you're asking whether the instruction to always mention your question when the bot replies is part of the bot or if it comes from an external library. If I understand you correctly, the quoted message feature that prompts you to mention your question is actually from the `whatsapp-web.js` library.

[here are examples from `whats-web.js`](https://github.com/pedroslopez/whatsapp-web.js/blob/main/example.js)

if you want to change this you need WhatsApp client instance from [WhatsAppClient.ts.](https://github.com/Zain-ul-din/whatsapp-ai-bot/blob/master/src/lib/WhatsAppClient.ts)

![image](https://user-images.githubusercontent.com/78583049/222797286-cc4cd7da-f5dc-4717-aec3-3323334e6db1.png)

**Note!**
Currently, you can't access clients inside models to access clients in models you can make WhatsAppClientClass `Singleton.`

**Please create an issue if you want any features.**
